src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -233,6 +232,7 @@ struct FancyZones : public winrt::implements<FancyZones, IFancyZones, IFancyZone
 UINT FancyZones::WM_PRIV_VDCHANGED = RegisterWindowMessage(L"{128c2cb0-6bdf-493e-abbe-f8705e04aa95}");
 UINT FancyZones::WM_PRIV_VDINIT = RegisterWindowMessage(L"{469818a8-00fa-4069-b867-a1da484fcd9a}");
 UINT FancyZones::WM_PRIV_EDITOR = RegisterWindowMessage(L"{87543824-7080-4e91-9d9c-0404642fc7b6}");
+UINT FancyZones::WM_PRIV_VDUPDATE = RegisterWindowMessage(L"{b8b72b46-f42f-4c26-9e20-29336cf2f22e}");

```

Comment:
Same here.

Reply:
Moved, and rearranged a bit (init goes first, switch second, update third).